egs/multi_en/s5/conf/ami_ihm/mfcc.conf

```diff
@@ -0,0 +1,4 @@
+--use-energy=false
+--sample-frequency=16000
```

Comment:
WSJ, AMI, ICSI, Tedlium are 16khz, but fisher and switchboard are 8khz. Since most applications/research is done on the 8khz part, I'd suggest we rather limit the feature banks to 8khz, as opposed to upsampling this data to 16khz.